### ISSUE TITLE
HIGH and LOW functions don't work in all contexts

### ISSUE THREAD

**Issue by ekimekim** [rednex/rgbds#238]
Not entirely sure if this is a bug or feature request. Certainly it's confusing.

The `HIGH` and `LOW` builtin functions seem to not work when evaluating expressions needed during compilation. For example:
```
SECTION "Example", ROM0

MYCONST EQU $1234

; This works
db HIGH(MYCONST)

; This causes a syntax error
PRINTV HIGH(MYCONST)
```

In contrast, other functions such as `SIN()` do work in both contexts.

This is low priority since I can always use `x/256`/`x % 256` instead of `LOW`/`HIGH` but it's surprising and annoying since they work otherwise.

**Comment by AntonioND** [comment]
Yeah, I completely forgot about this case when I implemented it, thanks.
